src/staff.cpp

```diff
@@ -490,10 +490,10 @@ int Staff::ScoreDefOptimize(FunctorParams *functorParams)
     matchTypeLayer.ReverseComparison();
     this->FindAllDescendantsByComparison(&layers, &matchTypeLayer);
 
-    ListOfObjects mRests = this->FindAllDescendantsByType(MREST);
+    ListOfObjects notes = this->FindAllDescendantsByType(NOTE);
```

Comment:
If you are using notes as a condition, why would you look for all of them? Finding one will be enough.

Reply:
Right! Didn't think about that. Should be even faster.

Reply:
Thanks for the hint!